js/actors/arm.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Player class. Contains all movement logic and instance variables.
+ */
+class Arm extends Actor {
+
+    constructor(player, x = 0, y = 300, w = 20 * 0.8, h = 36 * 0.8) {
+        super(x, y, w, h, "armOpen");
+        this.hitboxes = [new Hitbox(this, 0, 0, this.width - 4, this.height - 4)];
+        this.animationFrameDuration = 20;
```

Comment:
we dont need to set animationframeduration or animationframes since there's just one image. super constructor will take care of doing this